pom.xml

```diff
@@ -13,6 +13,7 @@
     <module>kubernetes</module>
     <module>proto</module>
     <module>extended</module>
+    <module>fluent</module>
```

Comment:
I am removing it explicitly, because I don't want this to be done as part of the `compile` | `package` | `install` stages, I want it only to be run when we re-generate the Kubernetes client code from the OpenAPI spec.